src/__init__.py

```diff
@@ -17,3 +17,10 @@
     __all__.append('autoquality')
 except ImportError:
     pass
+
+
+try:
+    from src.integrations import clearml  # noqa: F401
```

Comment:
Please use relative import here
```suggestion
    from .integrations import clearml  # noqa: F401
```